### ISSUE TITLE
Be able to favorite within the establishment information page.  Now it has to be done in the “all” list.

### ISSUE THREAD

**Issue by BigActual** [disAbility-Connections/ion-ap3-child#5]

**Comment by d4mation** [comment]
This would end up being related to #2. As long as the code to allow adding to favorites doesn't only exist on the "all" list, I should hopefully be able to add a link within the Post Content to accomplish this.

**Comment by BigActual** [comment]
Favoriting is a slide action. correct? Can you confirm the code to allow adding favorites exists outside of the "all" list?

**Comment by d4mation** [comment]
It is a slide action on the "all" page, but if we were to put it on the Establishment page, I imagine it would need to be a link of sorts. Perhaps a little :star:. The difficult portion there would be ensuring that a state were available to show Favorite/Unfavorite.

It would likely be best to ask AppPresser support if something is possible, since it is somewhat surprising that a feature like this doesn't already exist.

**Comment by BigActual** [comment]
Just dotting the i. Are you submitting that request?

**Comment by BigActual** [comment]
can't we just put a star or something that you could click that would be a standardized item? We could add that on the WP side?

**Comment by d4mation** [comment]
No, because we have no way to trigger "Favoriting" a Post from within the Post Content. AppPresser does not expose that kind of functionality. We could add a :star:, but it wouldn't do anything.

It sounds like they're going to try to add that feature themselves, though.
